pkg/object/ks3.go

```diff
@@ -280,7 +280,7 @@ func newKS3(endpoint, accessKey, secretKey, token string) (ObjectStorage, error)
 		Endpoint:         strings.SplitN(uri.Host, ".", 2)[1],
 		DisableSSL:       !ssl,
 		HTTPClient:       httpClient,
-		S3ForcePathStyle: true,
+		S3ForcePathStyle: false,
```

Comment:
We still need this to be `true` for some existing KS3 clusters.

Reply:
Existing clusters are supported by both modes.

Reply:
No, some clusters on premise require S3ForcePathStyle to be true.

Reply:
Users can add *.xxx.com to enable vhost style for their existing clusters.